src/modules/disk_used.rs

```diff
@@ -0,0 +1,307 @@
+    } else {
+        format!("{}/{}", format_byte(used_space)?, format_byte(total_space)?)
+    };
+
+    let threshold_config = config.threshold_styles.iter().find(|threshold_style| {
+        (used_space / total_space) >= (threshold_style.threshold as f64 * 0.01f64)
+    });
+
+    let style = match threshold_config {
+        Some(threshold_config) => threshold_config.style,
+        None => config.default_style,
+    };
+
+    let parsed = StringFormatter::new(
+        "([$name:]($default_style))[$usage]($style)([$separator]($default_style))",
+    )
+    .and_then(|formatter| {
+        formatter
+            .map_style(|variable| match variable {
+                "style" => Some(Ok(style)),
+                "default_style" => Some(Ok(config.default_style)),
+                _ => None,
+            })
+            .map(|var| match var {
+                "name" if show_disk_name => match get_disk_name(disk) {
+                    Some(name) => Some(Ok(name)),
+                    _ => None,
+                },
+                "usage" => Some(Ok(formatted_usage.as_str())),
+                "separator" if add_separator => Some(Ok(config.separator)),
+                _ => None,
+            })
+            .parse(None)
+    });
+
+    match parsed {
+        Ok(parsed) => Ok(parsed),
+        Err(e) => Err(Box::new(e)),
+    }
+}
+
+fn should_display_disk(disk: &Disk, threshold: i64) -> bool {
+    let percentage = (disk.get_total_space() - disk.get_available_space()) as f64
+        / disk.get_total_space() as f64;
+
+    percentage >= (threshold as f64 * 0.01f64)
+}
+
+#[derive(Debug)]
+struct DiskNotFoundError {
+    path: PathBuf,
+}
+impl DiskNotFoundError {
+    fn new(path: PathBuf) -> Self {
+        DiskNotFoundError { path }
+    }
+}
+impl fmt::Display for DiskNotFoundError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "No disk found for path: {:?}!", self.path)
+    }
+}
+impl Error for DiskNotFoundError {}
+
+#[cfg(target_os = "windows")]
+fn get_drive_from_path<'a>(path: &PathBuf, disks: &'a [Disk]) -> Result<&'a Disk, Box<dyn Error>> {
+    use std::{ffi::OsString, io::Error, iter::once, os::windows::prelude::*};
+    use winapi::um::fileapi::GetVolumePathNameW;
+
+    let path: Vec<u16> = path.as_os_str().encode_wide().chain(once(0)).collect();
+    let mut volume_path_name: Vec<u16> = vec![0; 260];
+
+    let ret = unsafe {
+        GetVolumePathNameW(
+            path.as_ptr(),
+            volume_path_name.as_mut_ptr(),
+            volume_path_name.len() as u32,
+        )
+    };
+
+    if ret != 0 {
+        // Strip out the nulls from the end
+        while let Some(last) = volume_path_name.last() {
+            if *last == 0 {
+                volume_path_name.pop();
+            } else {
+                break;
+            }
+        }
+        let volume_path_name = OsString::from_wide(&volume_path_name);
+        log::info!("Got path name: {:?}", volume_path_name);
```

Comment:
The log level for this is too high.
```suggestion
        log::trace!("Got path name: {:?}", volume_path_name);
```